CHANGELOG.MD

```diff
@@ -7,6 +7,10 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## [Unreleased]
 
+## [1.0.2] - 2023-08-15
+- TMS-948: Style fixes: change contacts from 3 columns to 2, make video-block & component narrower, change button bg-color to red
```

Comment:
- TMS-948: 
  - Style fixes: change contacts from 3 columns to 2,
  - make video-block & component narrower
  - change button bg-color to red
  - Add divider-block & component
  - add exhibition-carousel component